engine/core/src/main/java/org/datacleaner/job/runner/ConsumeRowHandler.java

```diff
@@ -174,11 +174,14 @@ public ConsumeRowResult consumeRow(final InputRow row) {
 
         final RowProcessingPublisher publisher;
         if (rowConsumeConfiguration.table != null) {
-            publisher = rowProcessingPublishers.getRowProcessingPublisher(rowConsumeConfiguration.table);
-            if (publisher == null) {
+            @SuppressWarnings("deprecation")
+            RowProcessingPublisher tablePublisher = rowProcessingPublishers
```

Comment:
final missing. But why introduce the temp variable at all? Original seemed clearer.